ComponentKit/Core/Scope/CKComponentScopeHandle.mm

```diff
@@ -201,6 +201,16 @@ @implementation CKScopedResponder
   std::mutex _mutex;
 }
 
+- (instancetype)init
+{
+  if (self = [super init]) {
+    static CKScopedResponderUniqueIdentifier nextIdentifier = 0;
+    _uniqueIdentifier = OSAtomicIncrement32(&nextIdentifier);
```

Comment:
I guess this is maybe better than just using the pointer value of the responder, which would probably work as well without introducing as much code.